docs/server/backup.rst

```diff
@@ -0,0 +1,20 @@
+.. _backup:
+
+Backup your Pootle system
+=========================
+
+In particular you should backup:
+
+- All your translation files (your whole
+  :setting:`POOTLE_TRANSLATION_DIRECTORY`). Use the :djadmin:`sync_stores`
+  command to synchronize all your translation files to disk before making any
+  backup.
```

Comment:
We might want to make user use ``--force`` and ``--overwrite`` - though that might confuse things.